noir-projects/noir-protocol-circuits/src/crates/types/src/messaging/l2_to_l1_message.nr

```diff
@@ -0,0 +1,36 @@
+use crate::{address::EthAddress, constants::L2_TO_L1_MESSAGE_LENGTH, traits::{Deserialize, Empty, Serialize}};
+
+struct L2ToL1Message {
```

Comment:
This one gets a little strange when we also have 
```solidity
struct L2ToL1Msg {
    L2Actor: sender,
    L1Actor: recipient,
    bytes32: content,
}
```

I get that it is the only values that we are really controlling here, but just pointing it out that it can be confusing.

Reply:
"Addressed" this just by adding there comments in https://github.com/AztecProtocol/aztec-packages/pull/4512/commits/6d5c87b72f8e545bed662319544c118eea037ddc

I was considering renaming the thing in Noir and TS to L2ToL1Payload but then I would have to rename `new_l2_to_l1_msgs` and that would worse the situation.